bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -948,13 +949,19 @@ void initNative(String filename) {
 			}
 			Gdip.Bitmap_delete(bitmap);
 			if (status == 0) {
-				if (this.handle == 0) SWT.error(error);
+				if (handle == 0) SWT.error(error);
 			}
 		}
 	}
+	return handle;
 }
 
 long [] createGdipImage() {
+	return createGdipImage(this.getCurrentDeviceZoom());
+}
+
+long [] createGdipImage(Integer zoomLevel) {
```

Comment:
As we call it zoom or nativeZoom anywhere else, we should call it zoom here as well